package.json

```diff
@@ -15,11 +15,11 @@
     "start": "next start -p 8080",
     "bs": "next build && next start -p 3000",
     "lint": "npm-run-all lint:*",
-    "lint:js": "eslint -c .eslintrc.js \"**/*.{js,jsx}\"",
+    "lint:js": "prettier --check \"**/*.{js,jsx}\" & eslint -c .eslintrc.js \"**/*.{js,jsx}\"",
```

Comment:
Based on suggestions from eslint and prettier docs, we are now running prettier on its own instead as an eslint rule.  Conceptually eslint should now be solely linting for opinons about haw javascript is used, while prettier should be solely linting for Code Style (formatting in the file).